Android.mk

```diff
@@ -1,6 +1,16 @@
 LOCAL_PATH:= $(call my-dir)
 
+ifneq (,$(wildcard frameworks/av/media/mediaserver/Android.mk))
 DROIDMEDIA_32 := $(shell cat frameworks/av/media/mediaserver/Android.mk |grep "LOCAL_32_BIT_ONLY[[:space:]]*:=[[:space:]]*" |grep -o "true\|1\|false\|0")
```

Comment:
Merely pointing out that in libhybris we've been checking the other file https://github.com/libhybris/libhybris/pull/467/commits/514c446bebc371778589341a6a48120adf840f0f#diff-68374cc913792e3e5344339f0ceab7b8e26786e7955d0c0022a0b2eaa4332e2fR4

Reply:
The file used in libhybris "frameworks/av/media/libmediaplayerservice/Android.mk" was removed in Android 9 and there is a now frameworks/av/media/libmediaplayerservice/Android.bp) which uses the same logic used in newly added code in this commit. Not sure which file is more correct to check, from libmediaplayerservice or mediaserver.

Reply:
Based on grepping different Android source trees the mediaserver always seems to have 32-bit build enabled but in libmediaplayerservice the line is commented out in some bases.

Reply:
I'll create separate PR to libhybris which replaces libmediaplayerservice to mediaserver.